applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -312,6 +314,8 @@ def AssembleTestSuites():
     # SPRISM tests
     nightSuite.addTest(TSprismMembranePatchTests('test_execution'))
     nightSuite.addTest(TSprismBendingPatchTests('test_execution'))
+    # Explicit solid beam
+    nightSuite.addTest(TExplicitSolidBeam('test_execution'))
```

Comment:
how long does the test run in your machine?

Reply:
5 sec

Reply:
way too long ... remember that travis is very much slower than your machine